spec/log-window-tests.js

```diff
@@ -54,59 +67,79 @@ describe('Log window', () => {
   describe('Window handling', () => {
     it('opens the logging window', () => {
       EmeLogWindow.instance.open();
-      expect(window.open).toHaveBeenCalledWith(
-        'log.html', jasmine.any(String), jasmine.any(String));
+      expect(chrome.windows.create).toHaveBeenCalledWith(new Object({
+        url: 'log.html', type: 'popup', height: 600, width: 700
+      }));
     });
 
-    it('reports the logging window is open', () => {
-      mockWindow.closed = false;
-      EmeLogWindow.instance.open();
+    it('reports the logging window is open', async function (done) {
```

Comment:
`done()` is the old style for asynchronous tests before the `async` keyword.  If you have an `async` function, it takes no arguments, and implicitly returns a `Promise` which Jasmine will use to know when the test is done.

Please remove all instances of `done`.  Thanks!

Reply:
Good to know! Removed all `done` from the arg and the test body.